ceremonies/src/migrations.rs

```diff
@@ -220,4 +328,34 @@ mod test {
 			assert_err!(state, "too many attestations");
 		});
 	}
+	#[allow(deprecated)]
+	#[test]
+	fn migration_to_v2_works() {
+		new_test_ext().execute_with(|| {
+			StorageVersion::new(1).put::<Pallet<TestRuntime>>();
+			assert_eq!(StorageVersion::get::<Pallet<TestRuntime>>(), 1);
+			// Insert some values into the v0 storage:
+			let alice: AccountId = AccountKeyring::Alice.into();
+			let old_rep = v1::Reputation::VerifiedLinked;
+			let cid = CommunityIdentifier::from_str("111112Fvv9e").unwrap();
+
+			v1::ParticipantReputation::<TestRuntime>::insert(
+				(cid, 1),
+				alice.clone(),
+				old_rep.clone(),
+			);
+
+			// Migrate.
+			let state = v2::MigrateToV2::<TestRuntime>::pre_upgrade().unwrap();
+			let _weight = v2::MigrateToV2::<TestRuntime>::on_runtime_upgrade();
+			v2::MigrateToV2::<TestRuntime>::post_upgrade(state).unwrap();
+
+			// Check that all values got migrated.
+
+			assert_eq!(
+				crate::ParticipantReputation::<TestRuntime>::get((cid, 1), alice),
+				Reputation::VerifiedLinked(1)
+			);
```

Comment:
Mean the code is very simple, but we might want to test that the other reputation variants are unaltered.

Reply:
hmmm, can't we trust the compiler with that?
I get your point about test coverage, but in this case I think the compiler is restrictive enough

Reply:
only if you correctly implemented the match statement ;)